assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/aws-ebs-csi-driver/templates/node.yaml

```diff
@@ -0,0 +1,108 @@
+---
+# Node Service
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: ebs-csi-node
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: ebs-csi-node
+  template:
+    metadata:
+      labels:
+        app: ebs-csi-node
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: eks.amazonaws.com/compute-type
+                operator: NotIn
+                values:
+                - fargate
+      nodeSelector:
+        kubernetes.io/os: linux
+        beta.kubernetes.io/arch: amd64
+      hostNetwork: true
+      priorityClassName: system-node-critical
+      tolerations:
+        - operator: Exists
+      containers:
+        - name: ebs-plugin
+          securityContext:
+            privileged: true
+          image: amazon/aws-ebs-csi-driver:latest
```

Comment:
If we do use the upstream Helm chart this would be changed, but if we stick with this, we shouldn't use latest (see for example [this](https://vsupalov.com/docker-latest-tag/)), but a tagged release.